.github/actions/deploy-vercel/client.mjs

```diff
@@ -0,0 +1,169 @@
+// @ts-check
```

Comment:
for this and the other `mjs` files: what do they do? Did you write them, or copy them from somewhere?

Reply:
I wrote these, it's just a bit of JS acting as a bridge between the Vercel REST API and the GitHub Actions environment. I'll add a bit of documentation, but most of the API calls are so severely underdocumented that linking to the official docs wouldn't help anyone trying to understand it 😢